lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -370,14 +370,12 @@ class LowerXMRPass : public circt::firrtl::impl::LowerXMRBase<LowerXMRPass> {
     }
 
     LLVM_DEBUG({
-      for (auto I = dataFlowClasses->begin(), E = dataFlowClasses->end();
-           I != E; ++I) { // Iterate over all of the equivalence sets.
+      for (const auto &I :
+           *dataFlowClasses) { // Iterate over all of the equivalence sets.
```

Comment:
Mega-nit: Moving the commend _above_ may get `clang-format` to put this all one one line.

Reply:
yup that's correct